chapters/package-information.md

```diff
@@ -1417,6 +1417,54 @@ EXAMPLE 2 RDF: Property `spdx:attributionText` in class `spdx:Package`
 </Package>
 ```
 
+## 7.24 Package Purpose field <a name="7.24"></a>
+
+### 7.24.1 Description
+
+This field provides information about the purpose of the identified package. Package Purpose is intrinsic to how the package is being used rather than the content of the package. The options to populated this field are limited to:
+
+`APPLICATION` if the package is a software application;
+`FRAMEWORK` if the package is a software framework;
+`LIBRARY` if the package is a software library;
+`CONTAINER` if the package refers to a container image which can be used by a container runtime application;
+`OPERATING-SYSTEM` if the package refers to an operating system;
+`DEVICE` if the package refers to a chipset, processor, or electronic board;
```

Comment:
I'm not sure I understand the meaning of `DEVICE` as defined here. Is the intention that this would be used for:

1. files which are the actual embodiment of a chipset, etc. (such as PCB images, or VHDL files, etc.); or
2. files which are specific to a particular device (such as architecture-specific files in the Linux kernel sources)

I'm guessing the intention is 1 but not 2, but the current definition seems to potentially apply to either one... maybe consider clarifying whether that's the intended meaning.